shell/platform/darwin/macos/framework/Source/FlutterRenderingBackend.mm

```diff
@@ -0,0 +1,44 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterRenderingBackend.h"
+
+#import <QuartzCore/QuartzCore.h>
+
+@implementation FlutterRenderingBackend
+
++ (BOOL)renderUsingMetal {
+  if (@available(macOS 10.14, *)) {
```

Comment:
Per previous discussion, there are two aspects to enabling Metal:
- What OS version we (read: Skia) are comfortable enabling Metal on.
- Whether the current host hardware supports Metal (not all Macs that can run macOS 10.14 support Metal)

I'm not seeing any querying of the second here, which would make this logic incorrect. See also https://github.com/flutter/flutter/issues/76710